### PR TITLE
Heat map

### PR DESCRIPTION
**Pull request by jdmartinez1062**

### CODE REVIEW

**Comment by Ispirett** [comment]
# Revewing

File: index.HTML [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                                <h3 class="heat-1 big-white-title txt-center">Smashing Newsletter</h3>
+                                <div class="heat-2 big-img"></div>
+                                <div class="button-div-parent heat-3"><div class="heat-2 button-div">Button</div></div>
+                        </div>
+                        <div class="col-1-1 latest-art-div heat-5">
+                                <h6 class="heat-4 latest-art">SEE ALL LATEST ARTICLES</h6>
+                        </div>
+                </section>
+            </article>   
+        <footer class="heat-5">
```
Comment (Ispirett) [inline index.HTML:201]: Please add links layout you can use text, but this is for you to practice what you learnt before, you can use flex or floats.

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @jdmartinez1062 @mariobarrioss    Excellent Job so far on this project you are doing a great job :100: 
image. 

See comment below 

submit for another review when finished

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi team :raised_hand_with_fingers_splayed: nice job.

 However, there are no Linter checks in this project.

- [ ] Ensure your project passes the checks, 

#### After affecting the change  :point_up_2:, submit another form for code review. :muscle: 

### This project hasn't been approved :stop_sign:

**Review by whiz25 — APPROVED** [review]
## Good to go :clap: 
Success in your next projects. Please merge and submit the completion form.
Use ```Afani Martin``` under TSE's who reviewed this project when submitting the project completion form.

**Pull request merged**